client/components/gravatar-with-hovercards/recommended-blogs/index.jsx

```diff
@@ -23,7 +23,7 @@ function RecommendedBlogs( { userLogin } ) {
 		}
 	}, [ userLogin, recommendedBlogs, dispatch ] );
 
-	const recommendedBlogsLink = `/reader/list/${ userLogin }/recommended-blogs`;
+	const recommendedBlogsLink = `/reader/users/${ userLogin }/recommended-blogs`;
```

Comment:
When I'm already on the user profile and I hover over the gravatar, the "View all" link re-shuffles the list instead of loading it, since we're already there. Should the link be disabled on that page? Or the hover card?

Reply:
I see! So "view all" when already on recommended blogs doesn't close the item and makes the list shuffle. Also the main gravatar and name seem to do the same when on the main user profile route.

What I have done is created a `closeCard` function and applied it to the links. So that when a link is clicked the card is closed, even if it doesn't actually navigate or change pages.

So now while on recommended-blogs, "view all" will just close the card, and clear the view for seeing all the recommended blogs! 😆 (and similar for the other profile links)

What do you think about this?